install_config/topics/configuring_azure_objects.adoc

```diff
@@ -0,0 +1,115 @@
+////
+Module included in the following assemblies:
+
+install_config/configuring_azure.adoc
+////
+
+[id='configuring-azure-objects_{context}']
+= Configuring Microsoft Azure objects
+
+Integrating {product-title} with Microsoft Azure requires the following
+components or services to create a highly-available and full-featured
+environment.
+
+[IMPORTANT]
+====
+To ensure that the appropriate amount of instances can be launched, request an
+increase in CPU quota from Microsoft before creating instances.
+====
+
+A resource group::
+
+Resource groups contain all Microsoft Azure components for a deployment,
+including networking, load balancers, virtual machines, and DNS. Quotas and
+permissions can be applied to resources groups to control and manage resources
+deployed on Microsoft Azure. Resource groups are created and defined per
+geographic region. All resources created for an {product-title} environment
+should be within the same geographic region and within the same resource group.
+
+See link:https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-overview[Azure Resource Manager overview] for more information.
+
```

Comment:
I might nit that these should be _For more information, see..._ But, I don't have to.  ;-)